CONTRIBUTING.rst

```diff
@@ -135,6 +135,67 @@ The key aspects of a committer are:
 * Visibility in community discussions (dev mailing list, Slack and GitHub)
 * Testing Release Candidates
 
+Guidelines for promoting Committers to Airflow PMC
+---------------------------------------------------
+
+To become a PMC member the committers should meet all **general prerequisites**.
+Apart from that the person should demonstrate distinct **community involvement** or **code contributions**.
+
+Prerequisites
+==============
+
+* Has been a committer for at least 3 months
+* Is still active community member
+
+Community involvement
+======================
+
+* Visibility on discussions on the dev mailing list
+* Spreading the “Airflow” either:
+
+  * Talks at meetups, conferences, etc
+  * Creating content (videos, blogs etc)
```

Comment:
```suggestion
  * Creating content like videos, blogs, etc
```
Will be more consistent with previous bullet

Reply:
Updated in https://github.com/apache/airflow/pull/13090/commits/4bc8e4ef24368afc778f2dbfd2bc4056c84add8e